CMakeLists.txt

```diff
@@ -98,6 +98,7 @@ set(action_files
 rosidl_generate_interfaces(${PROJECT_NAME}
   ${msg_files}
   ${srv_files}
+  ${action_files}
```

Comment:
These actions won't compile on ROS2 Crystal patch 2

https://github.com/ros2/rosidl/issues/348

Reply:
temporarily commented out with a todo to restore this line once the underlying issue is fixed